### DISCUSSION TITLE
Is there a reason why Plausible doesn't have a "Pages Per Session" metric?

### DISCUSSION THREAD

**Discussion by kristjanmar** [plausible/analytics#2001] [Anything Else]
To me, one of the best indicators of engagement is pages per session. I find myself always calculating it manually by dividing total pageviews by unique visitors or entrances. Is there a reason why Plausible doesn't include this by default?

**Comment by metmarkosaric** [comment]
Thanks for the feedback @kristjanmar! We keep the dashboard as simple as possible so have only included the most important metrics. Would be nice to do something more in terms of engagement which could be the pageviews per visit or a metric along those lines in the future. There's this thread here on the idea: https://github.com/plausible/analytics/discussions/131

**Comment by kristjanmar** [comment]
That's understandable, but I would argue that pages / session is one of the core metrics that should be included. I think it is just as important as bounce rate and visit duration, if not more.

**Reply by jdemailas** [reply]
Agreed this a core metric, and think it would be very easy to implement?

**Comment by metmarkosaric** [comment]
Update: "Pages per session" is in the dashboard and it is called "Views per visit"
